src/respend/respendrelayer.cpp

```diff
@@ -86,7 +86,7 @@ void RespendRelayer::Trigger()
     if (!valid || !interesting)
         return;
 
-    RelayTransaction(respend, true);
```

Comment:
As I understand it, MakeTransactionRef creates a copy of respend and wraps it in a shared pointer so that it will be automatically deleted.  Is there any way to avoid this copy?  Can "respend" be converted to a TransactionRef?